src/coreclr/src/inc/cordebug.idl

```diff
@@ -6472,6 +6475,27 @@ interface ICorDebugHeapValue3 : IUnknown
     HRESULT GetMonitorEventWaitList([out] ICorDebugThreadEnum **ppThreadEnum);
 };
 
+/*
```

Comment:
Should we also update the documentation for EnableGCNotification to indicate that it is deprecated?

Reply:
Thanks